app-libs/exchange-oracle/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+use crate::error::Error;
+
+pub mod coingecko;
+pub mod error;
+
+pub trait ExchangeRateGet {
```

Comment:
Just a minor style issue: In case the trait has only one method, the trait should be named the same as the method. Also, with a verb, use a name that is like an English sentence, in this case `GetExchangeRate`.